### DISCUSSION TITLE
How can i sideload nightly build for OSMANDplus?

### DISCUSSION THREAD

**Discussion by cyberman54** [osmandapp/OsmAnd#17355] [Q&A] [answered]
While the current release 4.4.7 has the very annoying accidential map rotation bug, i want to use the nightly build, because it already has this bug patched. I'm using OSMANDplus. How can i update the plus app by the nightly build? I am on Android13. Thanks for any hints.

**Comment by dmpr0** [comment, marked as answer]
Hi! 
You can try to install OsmAnd+ Beta version, it updates weekly https://play.google.com/apps/testing/net.osmand.plus.
Unfortunately, you can not have installed both OsmAnd+ and Nightly+, because they both had the same package name: `net.osmand.plus`. You can install nightly version for regular OsmAnd.

**Reply by cyberman54** [reply]
Thank you very much for this useful hint! It solved my problem.
